modules/Bio/EnsEMBL/Analysis/Hive/Config/SanityChecksStatic.pm

```diff
@@ -140,6 +140,118 @@ sub _master_config {
    },
 
   'gene_db_checks' => {
+    'bird_basic' => {
+      'genblast' => {
+        'logic_names' => {
+          'genblast'            => 200000,
+          'genblast_not_best'   => 250000,
+        }, # logic_names
+        'biotypes' =>    {
+          'human_pe12_'         => 20000,
+          'aves_pe12_'          => 15000,
+          'mammals_pe12_'       => 100000,
+          'reptiles_pe12_'      => 1000,
+        }, # biotypes
+      }, # genblast
```

Comment:
is the ig-tr check missing for a reason? is there nothing for birds from this pipeline?

Reply:
There is nothing basically from the birds I checked. I think this must have been skipped by the pipeline or possibly not set. Even the Chicken @thibauthourlier worked on are missing this entry.